boot/seal.go

```diff
@@ -58,7 +75,23 @@ func recoveryBootChainsFileUnder(rootdir string) string {
 // sealKeyToModeenv seals the supplied keys to the parameters specified
 // in modeenv.
 // It assumes to be invoked in install mode.
-func sealKeyToModeenv(key, saveKey secboot.EncryptionKey, model *asserts.Model, modeenv *Modeenv) error {
+func sealKeyToModeenv(key, saveKey secboot.EncryptionKey, model *asserts.Model, bootWith *BootableSet, modeenv *Modeenv) error {
+	if HasFdeSetupHook != nil && HasFdeSetupHook(bootWith) {
```

Comment:
perhaps we should just by default set this to something that returns false ?

Reply:
Excellent suggestion, thank you!